www-apps/bugzilla/bugzilla-5.0.6.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit webapp eutils
```

Comment:
Can we alphabetically sort these?

Reply:
Damn repoman should catch all of these ;-)

Reply:
Thanks!